ds-caselaw-ingester/tests.py

```diff
@@ -135,6 +158,66 @@ def test_handler_messages_v2(
         assert "Upload Successful" in log
         assert "Ingestion complete" in log
 
+    @patch("lambda_function.api_client", autospec=True)
+    @patch("lambda_function.extract_metadata", autospec=True)
+    @patch("lambda_function.tarfile")
+    @patch("lambda_function.boto3.session.Session")
+    @patch("lambda_function.urllib3.PoolManager")
+    def test_handler_messages_s3(
+        self, urllib_pool, boto_session, tarfile, metadata, apiclient, capsys
+    ):
+        """Mostly intended as a very sketchy test of the primary function"""
```

Comment:
Could we clarify what exactly is being tested here. Maybe something like

Given a s3 message that looks like blah...
When this messages is published to sns topic X
Then [what I expect to happen]